code/datums/controllers/job_controls.dm

```diff
@@ -20,15 +20,16 @@ var/datum/job_controller/job_controls
 			new /datum/job/research/medical_doctor/derelict {limit = 6; name = "Salvage Medic";} (),
 			new /datum/job/engineering/engineer/derelict {limit = 6; name = "Salvage Engineer";} (),
 			new /datum/job/civilian/staff_assistant (),
-			new /datum/job/civilian/chef (),
-			new /datum/job/civilian/barman (),
+			new /datum/job/catering/chef (),
+			new /datum/job/catering/barman (),
```

Comment:
this would put chefs and barmen into the disaster mode spawns, making sure if this is intentional?

Reply:
They were already there IIRC, I was just updating the paths to avoid runtimes